docs/UserGuide.md

```diff
@@ -157,7 +157,7 @@ Example:
 Adds a cheese to CHIM.
 
 Format: `addcheese t/CHEESE_TYPE q/QUANTITY [d/MANUFACTURE_DATE] [e/EXPIRY_DATE]`
-* The specified `QUANTITY` must be a positive integer.
+* The specified `QUANTITY` must be a positive integer and less than 1000.
```

Comment:
If upper limit is 1000, this should be "less than or equal to" 1000?

Reply:
Modified in latest commit